doc/content2.tex

```diff
@@ -1,725 +0,0 @@
-
-        Pie Chart \ref{img:results-triple-pie} shows that Data Mining triple exponential smoothing produces lower MSE
-        in 67\% of all eighteen tests. It is better result than for simple or double exponential smoothing. However
-        three tests produced MSE higher than 20\%. This could happen for example because R may use some better strategy
-        for decomposing time series and setting seasonal indices.
-
-        \begin{figure}[H]
-            \begin{center}
-                \begin{tikzpicture}
-                    \pie[text=legend, radius=2, color={green!100 , green!50, yellow!50, red!80}]
-                    {67/<1,
-                    6/\interval[{1.01,1.05}],
-                    6/\interval[{1.05,1.10}],
-%                    11/ \interval[{1.1,1.2}],
-                    22/>1.2}
-                \end{tikzpicture}
-                \caption{The proportion of Data Mining MSE to R MSE ratios split into intervals.}
-                \label{img:results-triple-pie}
-            \end{center}
-        \end{figure}
-
-        Averages of the MSE ratios are showed in Chart \ref{img:results-triple-r}. From the chart it can be seen that
-        four out of nine tests produces higher MSE. R produces only 0.95 MSE in test case 200\,--\,12(12) on
-        \texttt{sine} sample, but for smaller training sizes it produces 4.31 and 2.26 which is quite unstable result.
-        Data Mining produces 4.07, 2.15 and 2.63 (training sets of size 50, 100 and 200). This causes significantly
-        higher ratio for this test case.
-
-        Overall results for averaged MSE ratios show that Data Mining performs better than R on smaller training sets.
-
-        \begin{figure}[H]
-            \begin{center}
-                \scalebox{0.65}[0.5]{\includegraphics[angle=0]{img/src/results-triple-r.pdf}}
-                \caption{Test results for triple exponential smoothing. Comparison to reference R model.}
-                \label{img:results-triple-r}
-            \end{center}
-        \end{figure}
-
-        Chart \ref{img:results-triple-mse} emphasize that larger training size does improve forecasting accuracy. It is
-        the same result as for other models.
-
-        \begin{figure}[H]
-            \begin{center}
-                \scalebox{0.65}[0.5]{\includegraphics[angle=0]{img/src/results-double-mse.pdf}}
-                \caption{Test results for triple exponential smoothing. Average values of MSE for all tests samples.}
-                \label{img:results-triple-mse}
-            \end{center}
-        \end{figure}
-
-        %%%%%%%%
-        \subsection{Overall Results} \label{sec:overall}
-        This section contains aggregated results for simple, double and triple exponential smoothing models.
-
-        Chart \ref{img:results-overall-r} highlight that only in four tests Data Mining models in average produces
-        higher MSE than alternative R models. Test case 200\,--\,12(12) shows the worst result which is 28\%
```

Comment:
Chart \ref{img:results-overall-r} highlights / 3.osoba

...only in four tests Data Mining Models in average produce /namiesto produces